slack_bolt/workflows/step/utilities/async_update.py

```diff
@@ -0,0 +1,14 @@
+from slack_sdk.web.async_client import AsyncWebClient
+
+
+class AsyncUpdate:
+    def __init__(self, *, client: AsyncWebClient, body: dict):
+        self.client = client
+        self.body = body
+
+    async def __call__(self, *, inputs: dict, outputs: list,) -> None:
+        await self.client.workflows_updateStep(
+            workflow_step_edit_id=self.body["workflow_step"]["workflow_step_edit_id"],
+            inputs=inputs,
```

Comment:
TODO: `inputs` and `outputs` aren't required here. Also possible to pass in additional args